kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/ConsumerSeekUtil.java

```diff
@@ -72,14 +72,14 @@ public static void seekToCurrentPositions(Consumer<?, ?> consumer, Supplier<Kafk
     /**
      * Get all the {@link TopicPartition topicPartitions} belonging to the given {@code topics}.
      *
-     * @param consumer a Kafka {@link Consumer}
-     * @param topics   a list with topics
+     * @param consumer      a Kafka {@link Consumer}
+     * @param subscriber    a {@link KafkaSubscriber} that contains the topics to subscribe to.
      * @return a list of all the {@link TopicPartition topicPartitions}
      */
-    public static List<TopicPartition> topicPartitions(Consumer<?, ?> consumer, List<String> topics) {
+    public static List<TopicPartition> topicPartitions(Consumer<?, ?> consumer, KafkaSubscriber subscriber) {
```

Comment:
This change is, sadly enough, a breaking change for users invoking this public method.
Hence, it would be better to add another method using the `KafkaSubscriber`, keeping the original signature intact.

Reply:
Ok, I brought back the original method and made and overload for the new call. I think that should be ok, but wanted to double check.

Reply:
Yep, that's great, thanks!